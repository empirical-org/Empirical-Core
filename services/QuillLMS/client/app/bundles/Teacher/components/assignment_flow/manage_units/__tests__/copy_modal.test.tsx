import { mount } from 'enzyme';
import * as React from 'react';

import CopyModal from '../copy_modal';

describe('CopyModal component', () => {
  it('renders', () => {
    const wrapper = mount(<CopyModal attributeName="publish date" closeFunction={() => {}} copyFunction={() => {}} />)
    expect(wrapper).toMatchSnapshot()
  })
})
