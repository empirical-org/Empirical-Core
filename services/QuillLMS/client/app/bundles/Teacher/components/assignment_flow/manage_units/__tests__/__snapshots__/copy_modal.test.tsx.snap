// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CopyModal component renders 1`] = `
<CopyModal
  attributeName="publish date"
  closeFunction={[Function]}
  copyFunction={[Function]}
>
  <div
    aria-live="polite"
    className="modal-container copy-dates-modal-container"
    tabIndex={-1}
  >
    <div
      className="modal-background"
    />
    <div
      className="copy-dates-modal quill-modal modal-body"
    >
      <div
        className="top-section"
      >
        <h1>
          Copy 
          publish date
           to all activities?
        </h1>
        <p>
          Copying will overwrite the settings you previously entered. Please confirm that you want to continue.
        </p>
      </div>
      <div
        className="button-section"
      >
        <button
          className="quill-button medium secondary outlined focus-on-light"
          onClick={[Function]}
          type="button"
        >
          Cancel
        </button>
        <button
          className="quill-button medium primary contained focus-on-light"
          onClick={[Function]}
          type="button"
        >
          Yes, copy
        </button>
      </div>
    </div>
  </div>
</CopyModal>
`;
